lamindb/_curate.py

```diff
@@ -1257,6 +1308,43 @@ def _log_mapping_info():
         return False, non_validated
 
 
+def standardize_categories(
+    values: Iterable[str],
+    field: FieldAttr,
+    using_key: str | None = None,
+    organism: str | None = None,
+    source: Record | None = None,
+):
```

Comment:
typehint return please and consider adding a docstring. This is not an underscore function and has many lines of code. It'd deserver at least a 1 liner.